lib/active_resource/base.rb

```diff
@@ -1126,7 +1127,18 @@ def last(*args)
       # This is an alias for find(:all). You can pass in all the same
       # arguments to this method as you can to <tt>find(:all)</tt>
       def all(*args)
-        WhereClause.new(self, *args)
+        if find_all_lazy_load
+          WhereClause.new(self, *args)
+        else
+          ActiveResource.deprecator.warn(<<~MSG)
+            In the next Active Resource release, calling ActiveResource::Base#all will defer the http request submission
```

Comment:
The problem is that we already released it, so the new behavior is already out there. I'd just document it and leave it be. I don't think people need to disable. It is easy to force the clause to execute calling `to_a` or `load` in the object.